### ISSUE TITLE
In Your Face Baby

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments#113]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/in-your-face-baby/

**Comment by angellisy** [comment]
Não sei o que deu em mim pra assistir a essa série

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/in-your-face-baby/#comment_1114017472>):

> Não sei o que deu em mim pra assistir a essa série

Concordo. A FOX vendeu ela tão bem pelos primeiros episódios que eu me convenci que seria boa ideia assistir, mesmo detestando musicais.



Eu respeito a opinião de quem gostou, claro, mas, pra mim, foi sofrível. Roteiro fraco e, em certos momentos, ridículos — tipo aquela coisa com o Sam e o Evan "se desencontrando", por exemplo. Atuações exageradas e uma quantidade quase interminável de músicas. Eu sei que a série tem uma temática musical, mas a sensação que eu tive foi de "Deu *block* no roteirista? Música! A cena editada ficou mais curta do que quando gravada? Música! Uma das músicas escolhidas não era longa o bastante? Corta a cena pra outros atores, coloca 20s de conversa e coloca outra música!" :open\_mouth:
